components/header/index.tsx

```diff
@@ -5,9 +5,17 @@ import Navbar from './navbar'
 import SideMenu from './navbar/SideMenu'
 import { NavbarLink } from '@lib/strapi/navbar'
 import ExpandableNavbar from './navbar/ExpandableNavbar'
+import SearchOverlay from './navbar/searchpage'
 
-const Header = ({ navbarLinks }: { navbarLinks: NavbarLink[] }) => {
+const Header = ({
+  navbarLinks,
+  sessionToken,
+}: {
+  navbarLinks: NavbarLink[]
+  sessionToken?: string
```

Comment:
This token could also be passed to LoginButton (via Navbar), would make it possible to simplify the logic checking if the user is logged in without using a useEffect